docs/hazmat/primitives/asymmetric/ec.rst

```diff
@@ -611,6 +611,12 @@ Key Interfaces
             :func:`~cryptography.hazmat.primitives.asymmetric.utils.decode_dss_signature`,
             which returns the decoded tuple ``(r, s)``.
 
+    .. attribute:: curve
+
+            :type: :class:`EllipticCurve`
```

Comment:
This should all be intended one amount less.

Reply:
Ah! Sorry about the delay on fix. Taken care of.